osu.Desktop/DiscordRichPresence.cs

```diff
@@ -108,16 +117,23 @@ private void updateStatus()
                             Url = $@"{api.WebsiteRootUrl}/beatmapsets/{beatmap.BeatmapSet?.OnlineID}#{ruleset.Value.ShortName}/{beatmap.OnlineID}"
                         }
                     };
+
+                    presence.Assets.LargeImageKey = default_image_key;
+                    if (beatmapSetOnline.Value != null && beatmapSetOnline.Value.Covers.List != null && Encoding.UTF8.GetByteCount(beatmapSetOnline.Value.Covers.List) <= 256) // Ensure the URL will fit and not throw.
```

Comment:
Code analysis says the `beatmapSetOnline.Value.Covers.List != null` check is redundant, so either it is redundant, or nullability annotations are wrong.

Reply:
I was originally using the smaller thumbnail but that seems to be null. GetByteCount will throw if a null is passed through, so the check before hand is to ensure safety.